ppr-ui/src/components/search/BusinessSearchAutocomplete.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <v-card
-    v-if="searchValue.length >= 3 && !searching"
+    v-if="searchValue && searchValue.length >= 3 && !searching"
```

Comment:
would it be possible to use something like `searchValue?.length` here instead?

Reply:
That would be nice wouldn't it!
Optional Chaining is a TypeScript feature which is not supported in the Vue template itself.

There might be a way but i'm not sure if its related to our vue version or what. We've short circuited instead for some time, might be worth looking at if this is possible!

Reply:
Was curious, a quick bit of research mentions support for this (typescript in the tempalte) in Vue 3 👍

Reply:
Neat! Ok thanks for the clarification